R/antigen.garnish_predict.R

```diff
@@ -1810,8 +1809,43 @@ if (assemble & input_type == "peptide"){
 
     dt <- dt[pep_mut %like% "^[ARNDCQEGHILKMFPSTWYV]+$"]
 
+    if ("pep_wt" %chin% names(dt)){
+
+      message("Checking for non-standard AA one-letter codes in \"pep_wt\". Non-standard rows will be discarded.")
+
+      dt <- dt[pep_wt %like% "^[ARNDCQEGHILKMFPSTWYV]+$"]
```

Comment:
@leeprichman why is this repeated inside this loop?

Reply:
same function on different columns, as discussed